pkg/spec/tf_job.go

```diff
@@ -91,6 +92,8 @@ type TfReplicaSpec struct {
 	// TfPort is the port to use for TF services.
 	TfPort        *int32 `json:"tfPort,omitempty" protobuf:"varint,1,opt,name=tfPort"`
 	TfReplicaType `json:"tfReplicaType"`
+	//TfVersion is only used when TfReplicaType == PS to automatically start a PS server
+	TfVersion string `json:"tfVersion,omitempty"`
```

Comment:
Can a TfJobSpec just have an optional TfImage? This would be a Docker Image URI for a container with default binaries. Right now we have two "tensorboard" and "grpc_std_server"?

Now that I'm looking at the code I think its better if we can avoid adding a layer of indirection in the form of a mapping from Tf versions to Docker images.